### DISCUSSION TITLE
회의실 의자에 앉아 있는게 꼭 각잡고 반성하고 있는 자세 같아요

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#9] [제보하기]
사용하는데 문제가 있는 건 아닙니다. 
그냥 봤는데 너무 웃겨서 제보합니다.

회의실 의자에 앉아 있는게 꼭 각잡고 반성하고 있는 자세 같아요. ㅋ

**Comment by shlee303** [comment]
의견 감사합니다! 동의하는 부분이라 액션 개선 예정입니다! 👍

**Comment by shlee303** [comment]
자세를 좀 더 자연스럽게 수정해보았습니다 !!! 검토 해봐 주시고 다른 의견도 언제든 추가로 부탁드려요~!!
